build.gradle

```
@@ -73,6 +73,7 @@ dependencies {
 
     // Jackson JSON processor
     api 'com.fasterxml.jackson.core:jackson-databind:2.16.1'
+    api 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
```

Can you please pin this dependency on version 2.16.1?

```suggestion
    api 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.16.1'
```